File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "fmt"

func Convert(number int) string {
	var result string

    if number % 3 == 0 {
        result = "Pling" 
    }
    
    if number % 5 == 0 {
        result += "Plang" 
    }

    if number % 7 == 0 {
        result += "Plong" 
    }

    if number % 3 > 0 && number % 5 > 0 && number % 7 > 0 {
        result = fmt.Sprint(number)
    }
    
    return result
}

File: solutions/csharp/raindrops/1/Raindrops.cs
using System.Text;

public static class Raindrops
{
    public static string Convert(int number)
    {
        var sb = new StringBuilder();
        if (number % 3 == 0) sb.Append("Pling");
        if (number % 5 == 0) sb.Append("Plang");
        if (number % 7 == 0) sb.Append("Plong");
        if (number % 3 > 0 && number % 5 > 0 && number % 7 > 0) sb.Append(number);
        return sb.ToString();
    }
}